src/plugins/integrations/AWSIntegration.ts

```diff
@@ -161,7 +165,7 @@ class AWSIntegration implements Integration {
             }
           } else if (serviceName === 'dynamodb') {
             const statementType = DynamoDBRequestTypes[operationName];
-            const tableName = statementType ? Utils.getDynamoDBTableName(request) : AWS_SERVICE_REQUEST;
+            const tableName = koalas(Utils.getDynamoDBTableName(request), AWS_SERVICE_REQUEST);
```

Comment:
If table name is not valid for the request, it is ok to use `AWS_SERVICE_REQUEST ` as the default span name, but it should not be used for `AwsDynamoTags.TABLE_NAME` tag. So you may handle this by introducing another local variable for span name.

Reply:
Additionally, as far as I see, currently `Utils.getDynamoDBTableName(request)` always return a non empty value

Reply:
Done. Introduced a `spanName` variable. `Utils.getDynamoDBTableName(request)` returns null now.